ui/app/components/ui/popover/popover.header.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const PopOverheader = ({ title, onClose }) => (
+  <div className="popover-header">
+    <h2 className="popover-header__heading">{title}</h2>
+    <a href="#" className="popover-header__close" onClick={onClose}>
+      <div className="popover-header__close--wrap">
```

Comment:
Nit: `wrap`, `line-a`, and `line-b` are not modifiers - they're elements. So the proper BEM classname would be `popover-header__close-wrap` here, for example